frontend/src/pages/AddSighting/AddSighting.tsx

```diff
@@ -82,6 +91,14 @@ export function AddSighting(): JSX.Element {
               <SpeciesDropdown getSpeciesIdFromDropdown={getSpeciesIdFromDropdown} />
             </div>
           </div>
+          <div className="form-group row justify-content-center pb-4">
+            <label htmlFor="areaLocation" className="col-lg-3 col-sm-2 col-form-label">
+              Get Lat & Long from AreaLocation:
```

Comment:
Could this be changed to Area/Location...?

Reply:
done